src/sarif/custom-sarif-types-21.ts

```diff
@@ -5,42 +5,19 @@ export const enum SarifLogVersion21 {
 }
 
 export namespace Result {
-    export const enum level {
+    export const enum kind {
```

Comment:
Is it necessary for us to use our own custom sarif types? I thought the official sarif types included kind/level enums already that we could reuse

Reply:
"Custom sarif types" might not be the right name for this file because it's more for declaring the values for the `kind` and `level` types (we can't assign a `type` to a value when we are using them).